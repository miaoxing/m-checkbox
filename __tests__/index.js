import {Checkbox, CheckboxGroup} from '..';
import {render} from '@testing-library/react';

describe('Checkbox', () => {
  test('Checkbox', () => {
    const {container} = render(<Checkbox/>);
    expect(container).toMatchSnapshot();
  });

  test('CheckboxGroup', () => {
    const {container} = render(<CheckboxGroup/>);
    expect(container).toMatchSnapshot();
  });
});
