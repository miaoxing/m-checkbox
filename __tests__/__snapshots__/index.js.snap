// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox Checkbox 1`] = `
<div>
  <taro-checkbox-core />
</div>
`;

exports[`Checkbox CheckboxGroup 1`] = `
<div>
  <taro-checkbox-group-core />
</div>
`;
